config/environments/development.rb

```diff
@@ -68,4 +68,45 @@
     domain: 'mg.idseq.net'
   }
   ActiveRecordQueryTrace.enabled = true
+
+  # Development logging configuration
+  logger           = ActiveSupport::Logger.new(STDOUT)
+  logger.formatter = config.log_formatter
+  config.logger    = ActiveSupport::TaggedLogging.new(logger)
+  config.log_to = %w[stdout file]
```

Comment:
Why do we need both ?

Reply:
Ah you mean stdout and file? or you mean `logger` and `config.logger`?

Reply:
stdout and file

Reply:
Ah it was in the sample code but I figured this way people can keep their development.log if they like?

Reply:
but open to removing it

Reply:
Not very important. I approved as is... was just curious if there is any particular reason as one can always tail the file. 
But I guess it can make it easy if you run the server in the foreground.